.tours/5---create unlocked packages.tour

```diff
@@ -0,0 +1,65 @@
+{
+  "title": "5 - Create Unlocked Packages",
+  "steps": [
+    {
+      "file": "sfdx-project.json",
+      "description": "`sfdx-project.json` file provides packaging-specific configuration options to support package creation",
+      "line": 1
+    },
+    {
+      "file": "sfdx-project.json",
+      "description": "`packageDirectories` lists all the directories used in package creation and source sync between the orgs and local directory.",
+      "line": 2
+    },
+    {
+      "file": "sfdx-project.json",
+      "description": "`path` attribute maps the package directory to the local directory relative path. All the Metadata for the ***ESObjects*** package is present in the `./es-base-objects` path",
+      "line": 4
+    },
+    {
+      "file": "sfdx-project.json",
+      "description": "`package` attribute maps path of local directory to the name of the package.",
+      "line": 5
+    },
+    {
+      "file": "sfdx-project.json",
+      "description": "Dependencies are configured between the packages using `dependencies` attribute. ***ESSpaceMgmtLWC*** package is dependent on the ***ESObjects***, ***ESBaseCodeLWC*** and ***ESBaseStylesLWC*** packages",
+      "line": 36
+    },
+    {
+      "file": ".github/workflows/packaging-sfdx-ci-develop.yml",
+      "description": "Creation of package versions for each package is automated using GitHub actions. First a package is created for each directory manually using `sfdx force:package:create` command. The name of the package is used to create package versions.",
```

Comment:
May be show an example command for one package.

Reply:
added example. The examples are also in the Trailhead Module as well. I will link it once the module is available!